src/sentry/receivers/outbox/control.py

```diff
@@ -59,37 +71,85 @@ def process_api_application_updates(object_identifier: int, region_name: str, **
-            )
-            logger.info(
-                "webhook_proxy.complete",
-                extra={
-                    "status": getattr(
-                        response, "status_code", 204
-                    ),  # Request returns empty dict instead of a response object when the code is a 204
-                    "request_path": webhook_payload.path,
-                    "request_method": webhook_payload.method,
+    try:
+        client = RegionSiloClient(region=region)
+        response = client.request(
+            method=webhook_payload.method,
+            path=webhook_payload.path,
+            headers=webhook_payload.headers,
+            # We need to send the body as raw bytes to avoid interfering with webhook signatures
+            data=webhook_payload.body,
+            json=False,
+            prefix_hash=sha1(f"{shard_identifier}{object_identifier}".encode()).hexdigest(),
+        )
+        logger.info(
+            "webhook_proxy.complete",
+            extra={
+                "status": getattr(
+                    response, "status_code", 204
+                ),  # Request returns empty dict instead of a response object when the code is a 204
+                "request_path": webhook_payload.path,
+                "request_method": webhook_payload.method,
+            },
+        )
+    except SiloClientError as e:
+        sentry_sdk.capture_exception(e)
+    except ApiHostError as err:
+        with sentry_sdk.push_scope() as scope:
+            scope.set_context(
+                "region",
+                {
+                    "name": region.name,
+                    "id": region.category,
+                    "address": region.address,
                 },
             )
-        except ApiConflictError as e:
-            logger.warning("webhook_proxy.conflict_occurred", extra={"conflict_text": e.text})
+            err_cause = err.__cause__
+            if err_cause is not None and isinstance(err_cause, RestrictedIPAddress):
+                # Region silos that are IP address restricted are actionable.
+                silo_client_err = SiloClientError("Region silo is IP address restricted")
+                silo_client_err.__cause__ = err
+                sentry_sdk.capture_exception(silo_client_err)
+                return
+            sentry_sdk.capture_exception(err)
+        return
+    except ApiTimeoutError as err:
+        raise err
+    except ApiConnectionResetError as err:
+        raise err
+    except ApiError as api_err:
+        err_cause = api_err.__cause__
+        if err_cause is not None and isinstance(err_cause, HTTPError):
+            orig_response: Response | None = err_cause.response
+            if (
+                orig_response is not None
+                and status.HTTP_500_INTERNAL_SERVER_ERROR <= orig_response.status_code < 600
+            ):
+                # Retry on 5xx errors
+                raise api_err
+        sentry_sdk.capture_exception(api_err)
```

Comment:
Do we want to capture these errors? This could include BadRequest/Conflict errors from jira and msteams when JWTs expire.

Reply:
You're right. I removed it and added a comment.